src/app/(admin)/admin/_utils/common.js

```diff
@@ -48,9 +48,12 @@ function transformAddresses({ addresses, type = 'create' }) {
 export const transformSupportFocuses = ({ focuses, focusesIds }) => {
   const focusesToUpdate = [];
   const focusesToCreate = [];
-  const focusesToDelete = toConnectList(focusesIds.filter(cutId => !focuses.some(focus => focus.id === cutId)));
 
-  focuses.forEach(focus => {
+  const focusesToDelete = toConnectList(
+    focusesIds.filter(cutId => !focuses?.some(focus => focus.id === cutId)) ?? true,
```

Comment:
`focusesIds.filter(...)) ?? true` Do you think this makes any sense?
As I know the ["nullish coalescing operator"](https://uk.javascript.info/nullish-coalescing-operator) works with `null/undefined`, but [.filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter#return_value) always returns an array (empty or not)

Reply:
Thanks for comment! Sorry, just typo here, should be like `!focuses?.some(focus => focus.id === cutId) ?? true`.